src/main/java/com/spotinst/sdkjava/model/bl/ocean/kubernetes/ClusterIamInstanceProfileSpec.java

```diff
@@ -13,6 +13,7 @@
     @JsonIgnore
     private Set<String> isSet;
     private String      arn;
+    private String      name;
 
     private ClusterIamInstanceProfileSpec() {
```

Comment:
Shouldn't this class be with with Vng naming convention.

Reply:
I reuse the class from the cluster creation, will create another one for VNG